ig-data/ig.ini

```diff
@@ -1,7 +1,6 @@
 [IG]
 ig = input/ImplementationGuide-fhir.us.mcode.json
 template = hl7.fhir.template#0.0.5
```

Comment:
Maybe it's my setup, but I needed to re-add `template = hl7.base.template#current` to get the IG Publisher to run. Not a blocker though since I know to add it as a workaround.

Reply:
That's really odd, because the template documentation doesn't say you can specify two templates, and what happens if you do is totally undefined (from a spec perspective).  Also, it works for me without it.